oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/AsyncIndexUpdate.java

```diff
@@ -1050,6 +1051,7 @@ public void done(String now) {
         }
 
         public void failed(Exception e) {
+            boolean isConcurrentUpdateException = (e.getMessage() != null) && (e.getMessage().startsWith(CONCURRENT_EXCEPTIPN_MSG));
```

Comment:
what about adding "e instanceof ConcurrentUpdateException" ?

Reply:
I would make it multi-line, as the line is quite long.

Reply:
e instanceof ConcurrentUpdateException -> Actually it's not ConcurrentUpdateException but rather CommitFailedException - see https://github.com/apache/jackrabbit-oak/blob/trunk/oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/AsyncIndexUpdate.java#L1557#L1559 

And CommitFailedException is more generic so I didn't want to make use of instanceOf.

I will make it multi line though.